ofrak_core/setup.py

```diff
@@ -19,6 +19,15 @@ def run(self):
     long_description = f.read()
 
 
+entropy_so = setuptools.Extension(
```

Comment:
Any way to pass other compiler flags? If so, should we add stack protection? O3?

Reply:
There is, I think we should probably avoid those as they might be less portable between compilers. O3 is probably alright.